rs/embedders/benches/heap.rs

```diff
@@ -146,34 +189,44 @@ fn bench(c: &mut C, mem: Mem, call: Call, op: Op, dir: Dir, size: Size, step: St
     let throughput = throughput(size, step);
     let setup_action = setup_action(src);
 
-    let op = heap_op(op);
     let func_init_body = heap_func_init_body(mem, size, src);
-    let loop_body = loop_body(&op, mem, dir, size, step);
+    let op = heap_op(op, mem);
+    let loop_body = loop_body(&op, dir, 0, size, step);
     let canister_init_body = heap_canister_init_body(mem, size, src);
```

Comment:
Why can't we replace the init function by calling the setup function right after deploying the benchmark canister?

Reply:
There was a leftover from experiments. Those functions are different. I've fixed them and added some comments to clarify.